pandas/io/tests/parser/common.py

```diff
@@ -606,6 +606,21 @@ def test_multi_index_no_level_names(self):
         expected = self.read_csv(StringIO(data), index_col=[1, 0])
         tm.assert_frame_equal(df, expected, check_names=False)
 
+    def test_multi_index_blank_df(self):
+        # GH 14545
+        data = """a,b
+"""
+        df = self.read_csv(StringIO(data), header=[0])
+        expected = DataFrame(columns=[('a',), ('b',)])
```

Comment:
That's what I thought too, but I got dtype matching errors when I did that. I'll try to figure out what's going on.